README.md

```diff
@@ -1,19 +1,307 @@
+├── 📂 src/                  # Scala/Java source code
+├── 📂 scripts/              # Utility scripts
+├── 📂 legacy-content-import/ # Legacy import tools
+├── 📄 riff-raff.yaml        # Deployment configuration
+├── 📄 cdk.json             # CDK configuration
+├── 📄 package.json         # Node.js dependencies
+├── 📄 build.sbt            # Scala build configuration
+└── 📄 .nvmrc               # Node.js version
+```
+
+## 🔧 Configuration
+
+### 🌐 Environment Variables
+
+| Variable | Description      | Example                         |
+|----------|------------------|---------------------------------|
+| `App`    | Application name | `manage-help-content-publisher` |
+| `Stack`  | Guardian stack   | `membership`                    |
+| `Stage`  | Environment      | `CODE` / `PROD`                 |
+
+### 🪣 S3 Buckets
+
+- **Content Bucket:** `manage-help-content` (configurable via CDK context)
+- **Deployment Bucket:** `membership-dist` (configurable via CDK context)
+
+### 🔐 IAM Permissions
+
+The Lambda functions have permissions to:
+
+- 📖 **Read** from deployment bucket
+- 📝 **Read/Write/Delete** from content bucket
+- 📋 **List** bucket contents
+
+## 📊 Monitoring & Alerts
+
+### 🚨 CloudWatch Alarms (PROD only)
+
+| Alarm                    | Threshold      | Action    |
+|--------------------------|----------------|-----------|
+| **Publisher 4xx Errors** | ≥ 5 errors/min | SNS Alert |
+| **Publisher 5xx Errors** | ≥ 5 errors/min | SNS Alert |
+| **Takedown 4xx Errors**  | ≥ 5 errors/min | SNS Alert |
+| **Takedown 5xx Errors**  | ≥ 5 errors/min | SNS Alert |
+
+### 📈 Metrics Dashboard
+
+Monitor your APIs through:
+
+- 📊 **CloudWatch Dashboards**
+- 🔍 **API Gateway metrics**
+- ⚡ **Lambda function metrics**
+- 🪣 **S3 access patterns**
+
+## 🤝 Contributing
+
+### 🔄 Development Workflow
+
+1. **🌿 Create feature branch** from `main`
+2. **💻 Make changes** with proper testing
+3. **✅ Run checks:** `pnpm package`
+4. **📝 Create Pull Request**
+5. **🤖 CI/CD runs** automatically
+6. **👥 Code review** by team
+7. **🚀 Merge** triggers deployment
+
+### 📝 Code Standards
+
+- **TypeScript** for CDK infrastructure
+- **Scala** for Lambda functions
+- **ESLint + Prettier** for code formatting
+- **Jest** for unit testing
+- **Guardian CDK** patterns and constructs
+
+## 🆘 Troubleshooting
+
+### 🔍 Common Issues
+
+| Issue                   | Solution                                        |
+|-------------------------|-------------------------------------------------|
+| **CDK synthesis fails** | Run `pnpm type-check` to find TypeScript errors |
+| **Tests failing**       | Check `pnpm test` output for specific failures  |
+| **Deployment errors**   | Check Riff-Raff logs and CloudWatch             |
+| **API errors**          | Monitor CloudWatch alarms and Lambda logs       |
+
+### 📞 Support
+
+- **📚 Documentation:** [RIFF-RAFF.md](./RIFF-RAFF.md)
+- **🐛 Issues:** GitHub Issues
+- **💬 Team Chat:** Guardian Slack channels
```

Comment:
```suggestion
```

We don't have Slack. Anyone internally would already know this and anyone viewing it publicly would not have access to our chat.

Reply:
> We don't have Slack. Anyone internally would already know this and anyone viewing it publicly would not have access to our chat.

Whoops I meant: `Anyone internally would already know about our internal chat channels...`